src/main/java/frc/team4159/robot/subsystems/Grabber.java

```diff
@@ -45,24 +48,24 @@ public void iterate() {
             return;
         }
 
-        if (oi.getSecondaryJoy().getRawButton(7)) {
-            intake();
-        } else if (oi.getSecondaryJoy().getRawButton(8)) {
-            outtake();
+        if (oi.getSecondaryJoy().getRawButton(Constants.BUTTON_IDS.INTAKE)) {
+            grabberIntake();
+        } else if (oi.getSecondaryJoy().getRawButton(Constants.BUTTON_IDS.OUTTAKE)) {
+            grabberOuttake();
         } else {
-            stop();
+            stopGrabber();
         }
     }
 
-    private void intake() {
+    private void grabberIntake() {
```

Comment:
it might be helpful for the subsystem method names to reflect what their purpose is (e.g. intaking cargo / hatch)

Reply:
so for `Nose` should i change the hooks solenoid method names to be `grabHatch()` and `releaseHatch()`?

Reply:
yeah that would be good, thanks!